### PR TITLE
Prac 6 code review request

### PR DESCRIPTION
**Pull request by RashmiPokuru**
Hi @zachmorse1,

Please do code review for all the files of prac 06 and share your feedback.

### CODE REVIEW

File: prac_06/guitars.py [hunk 1]
```diff
@@ -25,12 +26,15 @@ def main():
     maximum_cost_length = max(len(str(f"{guitar.cost:,.2f}")) for guitar in guitars)
     # print(maximum_name_length)
     # print(maximum_cost_length)
-    guitars.sort(key=attrgetter("year"))
-    for i, guitar in enumerate(guitars, 1):
-        vintage_display = " (vintage)" if guitar.is_vintage() else ""
```
Comment (zachmorse1) [inline prac_06/guitars.py]: I don't believe you need an else statement as it is a boolean value.

File: prac_06/guitars.py [hunk 2]
```diff
@@ -25,12 +26,15 @@ def main():
...
-        print(
-            f"Guitar {i}: {guitar.name:>{maximum_name_length}} ({guitar.year:{MAXIMUM_YEAR_LENGTH}}), "
-            f"worth $ {guitar.cost:>{maximum_cost_length},.2f}{vintage_display}")
+    if guitars:
+        guitars.sort(key=attrgetter("year"))
+        for i, guitar in enumerate(guitars, 1):
+            vintage_display = " (vintage)" if guitar.is_vintage() else ""
```
Comment (zachmorse1) [inline prac_06/guitars.py:32]: I don't believe you need an else statement as it is a boolean value.

File: prac_06/guitars.py [hunk 3]
```diff
@@ -2,8 +2,8 @@
 CP1404/CP5632 Practical - Playing the guitars
 """
 from operator import attrgetter
-
 from prac_06.guitar import Guitar
+MAXIMUM_YEAR_LENGTH = 4
```
Comment (zachmorse1) [inline prac_06/guitars.py:7]: Good use of constant!

File: prac_06/languages.py [hunk 4]
```diff
@@ -12,8 +12,12 @@ def main():
     visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
     print(python)
     languages = [python, ruby, visual_basic]
-    print(type(languages))
-    print(languages)
+    # print(type(languages))
+    # print(languages)
+    print("The dynamically typed languages are:")
+    for language in languages:
+        if language.typing == "Dynamic":
```
Comment (zachmorse1) [inline prac_06/languages.py]: Utilise the is_dynamic() method we created in ProgrammingLanguage class.

Comment (RashmiPokuru) [reply prac_06/languages.py]: That's right! I missed it. I changed the code now. Thanks!

File: prac_06/guitars.py (hunk 2 above)
Comment (RashmiPokuru) [reply prac_06/guitars.py:32]: I think else is required for ternary operator. If there is no else and guitar is not vintage, vintage_display variable is undefined (not an empty string). 
If I have initiated vintage_display = "" before if statement, then else is not required like below. What do you think?
vintage_display = ""
if guitar.is_vintage():
    vintage_display = " (vintage)"
(Ignore alignment. For reason, it is not showing up in comment)

**Pull request merged**